includes/admin/upgrades/upgrade-functions.php

```diff
@@ -1043,6 +1044,23 @@ function give_v20_upgrades_email_setting() {
 	}
 }
 
+/**
+ * Update Donor Register Email Message.
+ *
+ * @since 2.0
+ */
+function give_v20_upgrade_donor_register_email_message() {
+	$all_setting = give_get_settings();
+
+	// Bailout on fresh install.
+	if ( empty( $all_setting ) ) {
+		return;
+	}
+
+	$donor_register_email = new Give_Donor_Register_Email();
```

Comment:
@mehul0810 Use `get_instance` function here

Reply:
@ravinderk I've updated the code. Please review,.